package.json

```diff
@@ -45,7 +45,8 @@
     "homepage": "https://github.com/hashicorp/vault-action#readme",
     "dependencies": {
         "got": "^11.5.1",
-        "jsonata": "^1.8.2"
+        "jsonata": "^1.8.2",
```

Comment:
Were these updates needed or introduced by accident? If not we'll handle these updates.

Reply:
jsonata version remains the same as before, it's just comma in the end of the line.
jsrsasign is new library, for JWT tokens generation, that i choose from recommended libraries on https://jwt.io

Reply:
Oh my apologies 😄 , I see now. Thanks!